.github/workflows/ci.yml

```diff
@@ -11,16 +11,16 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/checkout@v4
+      - run: corepack enable
+      - uses: actions/setup-node@v4
         with:
           node-version: 18
-          cache: 'yarn'
-      - name: Install
-        run: yarn install
+          cache: 'pnpm'
+      - run: pnpm install
       - name: Check types
-        run: yarn check:types
+        run: pnpm run check:types
```

Comment:
This is intentional?

Reply:
I actually wanted to discuss about this. It was pretty handy to use `pnpm link` while testing this branch. And what if we replace yarn with pnpm in this project as well?

Reply:
I am fine with this change :) it probably will result in CI CD action changes as well.